### ISSUE TITLE
cli tutorial seems outdated and is very hard to follow

### ISSUE THREAD

**Issue by bjaanes** [ethereum/ethereum-org#314]
- Dead links to the downloads
- Outdated information for creating private testnets (http://ethereum.stackexchange.com/questions/125/how-do-i-set-up-a-private-ethereum-network)
- Little explanation of what many of the different parameteres does + all paths are mac specific (for instance: --ipcpath ~/Library/Ethereum/geth.ipc (I have no idea what this is, luckily found another tutorial that explained it better).

I think the dev tutorials need to be up to date and really good if developers are to come to this platform...

**Comment by 5chdn** [comment]
:+1: just read the cli page too and was surprised it was listing the c++ and python clients as reference. would be nice to swap them in favor of parity (rust) and harmony (java). and maybe link other clients in the remainder (but without using them as cli reference).

**Comment by 5chdn** [comment]
related #587

**Comment by alexvandesande** [comment]
The command line tutorial was written before we had many GUI apps available. I recommend begginners to start at other places
